exonum-java-binding/core/rust/src/storage/entry.rs

```diff
@@ -110,15 +99,11 @@ pub extern "system" fn Java_com_exonum_binding_core_storage_indices_EntryIndexPr
     entry_handle: Handle,
     value: jbyteArray,
 ) {
-    let res = panic::catch_unwind(|| match *handle::cast_handle::<IndexType>(entry_handle) {
-        IndexType::SnapshotIndex(_) => {
-            panic!("Unable to modify snapshot.");
```

Comment:
Is it correct that all error handling is moved inside the core?

Reply:
Yes